waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -97,6 +98,17 @@ alexGetByte (_, [], (currChar:remainingSource)) = case encodeChar currChar of
 alexInputPrevChar :: AlexInput -> Char
 alexInputPrevChar (prevChar, _, _) = prevChar
 
+-- | Tokenizes the given string. 
+alexScanTokens :: String -> Maybe [Parser Token]
+alexScanTokens str = go ('\n', [], str)
+  where
+    go input@(_, _, lexeme) = case alexScan input 0 of
+      AlexEOF -> Just []
+      AlexError _ -> Nothing
+      AlexSkip rest _ -> go rest
+      AlexToken rest len act -> (currToken:) <$> go rest
```

Comment:
Instead of `act`, let's use `mkToken`?

Reply:
Definitely, I forgot that I get to name this function (thought it was an outside thing).